src/compute/src/logging/initialize.rs

```diff
@@ -96,54 +94,72 @@ struct LoggingContext<'a, A: Allocate> {
     shared_state: Rc<RefCell<SharedLoggingState>>,
 }
 
+pub(crate) struct LoggingDataflows {
+    pub logs: BTreeMap<LogVariant, (TraceBundle, usize)>,
+}
```

Comment:
Since there is only a single dataflow index now, we don't need to mention it for each log.

```suggestion
pub(crate) struct LoggingDataflows {
    pub traces: BTreeMap<LogVariant, TraceBundle>,
    pub dataflow_index: usize,
}
```

Also removes the unnamed `usize`, which is nice.